velox/vector/VectorStream.h

```diff
@@ -33,6 +33,12 @@ struct IndexRange {
   vector_size_t size;
 };
 
+/// Serializer that can iteratively build up a buffer of serialized rows from
+/// one or more RowVectors.
+///
+/// Use successive calls to `append` to add more rows to the serialization
+/// buffer.  Then call `flush` to write the aggregate serialized data to an
+/// OutputStream.
```

Comment:
Can you change API to be deprecated as deprecatedXXX and remove in followup? Thanks!